src/main/java/com/valmiki/adhyapakservice/entity/Courses.java

```diff
@@ -0,0 +1,76 @@
+package com.valmiki.adhyapakservice.entity;
+
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = "courses")
+public class Courses {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private int id;
+
+    @Column(name = "course_name")
+    private String courseName;
+
+    @Column(name = "subheading")
+    private String subHeading;
+
+    public Courses() {
+
+    }
+
+    public Courses(int id, String courseName, String subHeading) {
```

Comment:
Why aren't we using lombok annotations to create constructor, getter, and setter.